modules/createHashHistory.js

```diff
@@ -210,10 +210,7 @@ function createHashHistory(props = {}) {
           pushHashPath(encodedPath);
 
           const prevIndex = allPaths.lastIndexOf(createPath(history.location));
-          const nextPaths = allPaths.slice(
-            0,
-            prevIndex === -1 ? 0 : prevIndex + 1
-          );
+          const nextPaths = allPaths.slice( 0,  prevIndex + 1 );
```

Comment:
See previous comment.

Reply:
If there is a new version of history, isn't this PR unnecessary?